src/newbank/server/database/TransactionDB.java

```diff
@@ -0,0 +1,49 @@
+package newbank.server.database;
+
+import newbank.server.model.Account;
+import newbank.server.model.Customer;
+import newbank.server.model.Transaction;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TransactionDB {
+  private ArrayList<Transaction> transactions;
+  private Long transactionId;
+
+  private TransactionDB(){
+    this.transactions = new ArrayList<>();
+    this.transactionId = 0L;
+  }
+  private static TransactionDB instance;
```

Comment:
Please be consistent on the order of field declaration; preferably, they are declared at the beginning, inside of a class.

Reply:
Made the change